src/lib/features/feature-toggle/feature-toggle-store.ts

```diff
@@ -723,13 +723,13 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
         return result?.potentially_stale ?? false;
     }
 
-    async setCreatedByUserId(batchSize: number): Promise<void> {
+    async setCreatedByUserId(batchSize: number): Promise<number> {
         const EVENTS_TABLE = 'events';
         const USERS_TABLE = 'users';
         const API_TOKEN_TABLE = 'api_tokens';
 
         if (!this.flagResolver.isEnabled('createdByUserIdDataMigration')) {
-            return;
+            return -1;
```

Comment:
Same suggestion
```suggestion
            return 0;
```

Reply:
I'll return undefined here instead as we discussed